internal/commands/app/diff_test.go

```diff
@@ -139,7 +135,61 @@ Removed Dependencies
   - debug@4.3.1
 Modified Dependencies
   * underscore@1.9.1 -> underscore@1.9.2
-`, out.String())
+`
+
+		t.Run("with include node modules set it should diff function dependencies", func(t *testing.T) {
+			out, ui := mock.NewUI()
+			cmd := &CommandDiff{diffInputs{LocalPath: "testdata/dependencies", IncludeNodeModules: true}}
+			assert.Equal(t, nil, cmd.Handler(nil, ui, cli.Clients{Realm: realmClient}))
+
+			assert.Equal(t, diffStr, out.String())
+		})
+
+		t.Run("with include package json set it should diff function dependencies", func(t *testing.T) {
+			out, ui := mock.NewUI()
+			cmd := &CommandDiff{diffInputs{LocalPath: "testdata/dependencies", IncludePackageJSON: true}}
+			assert.Equal(t, nil, cmd.Handler(nil, ui, cli.Clients{Realm: realmClient}))
+
+			assert.Equal(t, diffStr, out.String())
+		})
+	})
+
+	t.Run("should return error with include node modules and include package json are both set", func(t *testing.T) {
```

Comment:
we should duplicate this test for when `--include-dependencies` and `--include-package-json` are set as well

---

also this test is incorrect, just by looking at the actual asserted upon error.  the error this test case is describing originates from the inputs' `Resolve` function, not `cmd.Handler`

---

on 3rd thought, this _appears_ to want to be `should return error with include package json set and diff dependencies returns an error`.  in actuality, it's probably reasonable to combine all 3 of those sorts of tests into one test table as well, to help readability (they essentially aren't testing anything different, but it will be good coverage for us)